line-bot-model/src/main/java/com/linecorp/bot/model/event/AccountLinkEvent.java

```diff
@@ -18,51 +18,49 @@
 
 import java.time.Instant;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
 
 import com.linecorp.bot.model.event.link.LinkContent;
 import com.linecorp.bot.model.event.source.Source;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Value;
 
 /**
  * Event object for when a user has linked his/her LINE account with a provider's service account.
  * You can reply to account link events.
  */
-@Value
 @JsonTypeName("accountLink")
+@Value
+@Builder(toBuilder = true)
+@AllArgsConstructor
+@JsonDeserialize(builder = AccountLinkEvent.AccountLinkEventBuilder.class)
 public class AccountLinkEvent implements Event, ReplyEvent {
+    @JsonPOJOBuilder(withPrefix = "")
+    public static class AccountLinkEventBuilder {
+        // Providing builder instead of public constructor. Class bodies filled by lombok.
```

Comment:
I cannot understand what this comment mean to. `@AllArgsConstructor` creates a public constructor, isn't it?

`Class bodies filled by lombok.` -> `Class body is filled by lombok.`

Reply:
Thank you. Updated.